vtexdocs/RapiDoc
# Feature/code section

#### What is the purpose of this pull request?

To update the curl style and create the structure of the try it section.

#### What problem is this solving?

The style of the curl section doesn't match the expected design in Figma. In addition, the test page needs the try it section that contains the store URL variables and the auth section (with auth keys), after all, Rapidoc-mini use requires the URL params and keys input to generate the correct curl request.

[You can find more information here :book:](https://www.notion.so/vtexhandbook/Estilizar-code-curl-section-da-API-reference-31418fc60aef43fea492baa3ba9fc8af)

**In the future, we'll need to move the Try section to the right side of the page, as the prototype in Figma.**

#### How should this be manually tested?

Go to [this page](https://deploy-preview-73--elated-hoover-5c29bf.netlify.app/) and check:
- The try it section style compared with the [Figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=6309%3A379335) (ignoring the request indentation in Figma)
- The Base URL works, and autocomplete the final URL below.
- You can copy the generated URL
- After filling in the header auth and the URL variables, click on the TEST METHOD button and check if the generated curl request is correct.
- For any point, compare with the actual behavior on the respective page in [Developer portal](https://developers.vtex.com/vtex-rest-api/reference/get-to-know-vtex-apis)

| This PR | Developer portal |
| ----------- | ----------- |
| [Get Product and SKU IDs](https://deploy-preview-73--elated-hoover-5c29bf.netlify.app/docs/api-reference/catalog) | [DP Page](https://developers.vtex.com/vtex-rest-api/reference/catalog-api-get-products-skus) |
| [Cart simulation](https://deploy-preview-73--elated-hoover-5c29bf.netlify.app/docs/api-reference/checkout) | [DP page](https://developers.vtex.com/vtex-rest-api/reference/cartsimulation) |
| [Retrieve Token](https://deploy-preview-73--elated-hoover-5c29bf.netlify.app/docs/api-reference/antifraud) | [DP Page](https://developers.vtex.com/vtex-rest-api/reference/1retrievetoken) |
| [Get GiftCard Provider by ID](https://deploy-preview-73--elated-hoover-5c29bf.netlify.app/docs/api-reference/giftcard) | [DP Page](https://developers.vtex.com/vtex-rest-api/reference/getgiftcardproviderbyid) |

#### Screenshots or example usage

| Before | After |
| ----------- | ----------- |
|![image](https://user-images.githubusercontent.com/42784961/187752154-8548f624-0589-4735-b786-0d19e08282e2.png)|![image](https://user-images.githubusercontent.com/42784961/187751969-6b6c7479-03b5-4b4e-bb3a-5c572d2466ab.png)|

Comparing the actual try section with the expected on Figma:
| This PR | Figma |
| ----------- | ----------- |
|![image](https://user-images.githubusercontent.com/42784961/187752434-e5e43ac5-be80-4b40-a19c-228ac2dfdf2c.png)|![image](https://user-images.githubusercontent.com/42784961/187752521-06cb8f08-6826-4839-8ca6-c990e3da516f.png)|

https://user-images.githubusercontent.com/42784961/187749913-05ee2ba4-3826-4773-a246-9f9b2d9194b9.mp4

#### Types of changes
